src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,122 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private Node<K, V> nullKeyNode;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
```

Comment:
Use the diamond operator to ensure type safety when creating a new array for the table. It should be 'new Node<>' or 'new Node<K,V>[]'.